Charm/ApplicationCore.h

```diff
@@ -64,7 +64,7 @@ class ApplicationCore : public QObject
         ShowAndRaise
     };
     explicit ApplicationCore( TaskId startupTask, QObject* parent = nullptr );
-    ~ApplicationCore();
+    ~ApplicationCore() override;
```

Comment:
Not a fan of "override" for virtual dtor reimpls, all channel (@milianw, @dfaure ) agrees with me ;)